build_deploy.sh

```diff
@@ -99,4 +103,31 @@ make \
   internal_image_registry="quay.io" \
   image_repository="$QUAY_ORG/kas-fleet-manager" \
   docker/login \
-  image/push
\ No newline at end of file
+  image/push
+
+# create new relaase in managed-kafka-versions repo for kas-fleet-manager
+if [[ ! -z "$AUTHOR_EMAIL" ]] && [[ ! -z "$AUTHOR_NAME" ]] && [[ ! -z "$GITLAB_TOKEN" ]] && \
+[[ ! -z "$PASSWORD" ]] && [[ ! -z "$USERNAME" ]]; then
```

Comment:
just realised that those two values (`PASSWORD` and `USERNAME`) aren't used. I will remove them after the review is completed by David Conde